### PR TITLE
Sd crossingdetails redo

### PR DESCRIPTION
**Pull request by ifsimicoded**

### CODE REVIEW

File: frontend/src/stories/statusToggle.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import StatusToggle from 'components/Dashboard/CrossingListPage/CrossingListItem/StatusToggle'
-import * as statusConstants from 'components/Dashboard/CrossingListPage/CrossingListItem/StatusConstants'
+import StatusToggle from 'components/Dashboard/CrossingListPage/CrossingListItem/StatusToggle';
+import * as statusConstants from 'constants/StatusConstants';;
```
Comment (briaguya) [inline frontend/src/stories/statusToggle.js]: ;;

File: frontend/src/css/mixins/_colors.scss [hunk 2]
```diff
@@ -8,4 +8,6 @@ $color-gray-25:              darken($color-white, 25%);
 $color-gray-50:              darken($color-white, 50%);
 $color-gray-75:              darken($color-white, 75%);
 
-$border-color-light: $color-gray-15;
+$border-color-light:          $color-gray-15;
+$color-primary:               #0F0035;
+$color-highlight:             #0054FF;
```
Comment (briaguya) [inline frontend/src/css/mixins/_colors.scss]: newline

File: frontend/src/css/global/_layout.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  }
+  .p#{$name}--tb {
+    padding-top: #{$size}px;
+    padding-bottom: #{$size}px;
+  }
+  .p#{$name}--lr {
+    padding-left: #{$size}px;
+    padding-left: #{$size}px;
+  }
+}
```
Comment (briaguya) [inline frontend/src/css/global/_layout.scss]: newline

File: frontend/src/css/global/_forms.scss [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  height: 40px;
+  width: 100%;
+  font-size: $base-font-size;
+  text-indent: 5px;
+
+  &.input--lg {
+    font-size: $font-large;
+    font-weight: bold;  
+  }
+}
```
Comment (briaguya) [inline frontend/src/css/global/_forms.scss]: newline

File: frontend/src/css/global/_colors.scss [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+}
+.gray--25 { 
+  color: $color-gray-25;
+}
+.gray--50 { 
+  color: $color-gray-50;
+}
+.gray--75 { 
+  color: $color-gray-75;
+}
```
Comment (briaguya) [inline frontend/src/css/global/_colors.scss]: newline

File: frontend/src/components/Dashboard/CrossingDetailPage/CrossingDetails.scss [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+.CrossingDetails__communities button {
+  margin-right: 10px;
+  margin-bottom: 10px;
+
+  &:last-of-type{
+    margin-right: 0;
+  }
+}
```
Comment (briaguya) [inline frontend/src/components/Dashboard/CrossingDetailPage/CrossingDetails.scss]: newline

File: frontend/src/stories/statusToggle.js (hunk 1 above)
Comment (ifsimicoded) [reply frontend/src/stories/statusToggle.js]: We'll take care of all the newlines and this goofiness with Ben's linter implementation.

**Pull request merged**